### ISSUE TITLE
Getting a warning message for deprecated API

### ISSUE THREAD

**Issue by kesavkolla** [remkop/picocli#1895]
I am using spring boot 3.0 with picocli 4.7.0

When I run my console application I am getting the following errror:

```
2022-12-15T14:23:58.493-08:00  WARN 700001 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: picocli.spring.boot.autoconfigure.PicocliAutoConfiguration

```

**Comment by remkop** [comment]
Hi @kesavkolla have you tried doing what the warning says: 
Compile the affected code with '-parameters'?
What happens when you run your code after compiling with '-parameters'?

**Comment by kesavkolla** [comment]
Yes I tried with -parameters to compileJava. Here is the snipped from my gradle file

```
compileJava {
    options.compilerArgs << '-parameters'
}

```

I am still getting that warning message.

**Comment by remkop** [comment]
This is not a picocli issue.
Looking at [this StackOverflow issue](https://stackoverflow.com/questions/74600681/warning-printed-after-migrating-to-spring-boot-3-0-spring-integration-6-0), "this is seen as a bug and will be fixed in Spring Framework 6.0.3."

I don't believe there is anything that can be done in picocli to address this, so allow me to close this ticket.
